### DISCUSSION TITLE
A landing +Take off  A32NX   vibrations on touch down the wheels on the runway

### DISCUSSION THREAD

**Discussion by welyy55** [flybywiresim/aircraft#4083] [Ideas]
Always wish you happy times
------------------------------

**Here I wanted to share with you an idea that is not new, but within the physical realism of the landing and the touch of the wheels to the runway results: a balanced landing that generates vibrations according to the type of descent, such as hard bumping on the runway at high speed, or according to the touch of one or two wheels, the physical movement is more realistic,

Their effect is strong at the onset of contact, and gradually diminishes with the speed set by the pilot.

And it may be a single physical effect, so as not to delve into deeper programming, and to be more easy to enter and update in the future.

Also, when takeoff, the motors' driving force generates strong vibrations, as in a realistic situation, and the pilot feels them according to the choice of the assigned camera position.

I am not a programmer and I am not a developer. I liked to add this idea because it creates a beautiful realism that may be the most beautiful in terms of its physical effect.**

------------------------------
Sorry for the many words

I wish you quieter times.

**Comment by MapleNAV-TCASFAULT** [comment]
Unfortunately headshake in MSFS is not something that we do or understand to program as this is something with the base sim and cannot be modded, maybe a mod like Chaseplane can be ported over to be compatible with MSFS to create your needed effect.

**Reply by welyy55** [reply]
I thank you for this response,

Chaseplane is agreat , Because the old emulator lacks such software. He was doing his guest job in the previous simulator and he was respectable
 and I really regret that. I hope that Asobo will take into account such gaps. Thank you
